base_layer/wallet/src/text_message_service.rs

```diff
@@ -259,7 +255,7 @@ impl TextMessageService {
         if let Some((info, msg)) = incoming_msg {
             trace!(
                 target: LOG_TARGET,
-                "Text Message received with ID: {:?} from {:?} with message: {:?}",
+                "Text Message received with ID: {:?} from {} with message: {:?}",
                 msg.id.clone(),
                 msg.source_pub_key.to_hex(),
```

Comment:
You can drop the `.to_hex()` part now that #516 is merged, I think.

Reply:
Thanks - done :)